command/sidechain/whitelist/params.go

```diff
@@ -0,0 +1,62 @@
+package whitelist
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	sidechainHelper "github.com/0xPolygon/polygon-edge/command/sidechain"
+)
+
+const (
+	newValidatorAddressesFlag = "addresses"
+)
+
+var (
+	errNoNewValidatorsProvided = errors.New("no new validators addresses provided")
+)
+
+type whitelistParams struct {
+	accountDir             string
+	accountConfig          string
+	privateKey             string
+	jsonRPC                string
+	newValidatorAddresses  []string
+	supernetManagerAddress string
```

Comment:
Don't need this flag. Whitelisting is done on child chain, and we know the address of the contract, since it is deployed on genesis, on a hardcoded address.